examples/terraform/README.md

```diff
@@ -0,0 +1,95 @@
+## How-To: using Terraform to manage an EKS Anywhere (EKS-A) Cluster
+A minimal guide which lays out how you can use Terraform to manage and modify an EKS-A cluster. This guide is meant for illustrative purposes only and is not meant as a definitive approach to building production systems with Terraform and EKS Anywhere.
+
+At its heart, EKS-A is a set of Kubernetes CRDs which define an EKS-A cluster, 
+and a controller which moves the cluster to match these definitions. 
+These CRDs, and the EKS-A controller, live on the management cluster, or, 
+in the case of a stand-alone cluster, the workload cluster itself.
```

Comment:
i feel we gonna settle down a name for this "stand-alone management/workload cluster" across the board. kinda confusing to call it workload cluster (tho it is a workload cluster) and mix it up w workload cluster in management setting

Reply:
hm yeah that's a good point. I wasn't sure what to call it here. Maybe just 'self-managed cluster'?